pages/configuration/elasticsearch.rst

```diff
@@ -8,108 +8,77 @@ We strongly recommend to use a dedicated Elasticsearch cluster for your Graylog
++----------------------------------------------------+-----------+--------------------------------------------------------------+-----------------------------+
+| ``elasticsearch_discovery_filter``                 | String    | Filter by node attributes for the discovered nodes           | empty (use all nodes)       |
++----------------------------------------------------+-----------+--------------------------------------------------------------+-----------------------------+
+| ``elasticsearch_discovery_frequency``              | Duration  | Frequency of the Elasticsearch node discovery                | ``30s`` (30 Seconds)        |
++----------------------------------------------------+-----------+--------------------------------------------------------------+-----------------------------+
 
-  # Public IP address or host name of the Graylog node, accessible for the other Elasticsearch nodes
-  elasticsearch_network_host = 198.51.100.23
+Automatic node discovery
+^^^^^^^^^^^^^^^^^^^^^^^^
 
+.. caution:: Automatic node discovery will be available starting with Graylog 2.3.0-alpha.3/beta.1.
 
-Also make sure to configure `Zen unicast discovery <http://www.elastic.co/guide/en/elasticsearch/reference/2.3/modules-discovery-zen.html#unicast>`__ in
-the Elasticsearch configuration file by adding the ``discovery.zen.ping.multicast.enabled`` and ``discovery.zen.ping.unicast.hosts`` settings with the
-list of Elasticsearch nodes to ``elasticsearch.yml``::
-
-  discovery.zen.ping.multicast.enabled: false
-  discovery.zen.ping.unicast.hosts: ["es-node-1.example.org:9300" , "es-node-2.example.org:9300"]
-
-The Elasticsearch default communication port is *9300/tcp* (not to be confused with the HTTP interface running on port *9200/tcp* by default).
-
-The communication port can be changed in the Elasticsearch configuration file (``elasticsearch.yml``) with the configuration setting ``transport.tcp.port``.
-
-Last but not least, make sure that Elasticsearch is binding to a network interface that Graylog can connect to (see ``network.host`` and `Commonly Used Network Settings <https://www.elastic.co/guide/en/elasticsearch/reference/2.3/modules-network.html#common-network-settings>`_).
-
+Graylog uses automatic node discovery to gather a list of all available Elasticsearch nodes in the cluster at runtime and distribute requests among them to potentially increase performance and availability. To enable this feature, you need to set the ``elasticsearch_discovery_enabled`` to ``true``. Optionally, you can define the a filter allowing to selectively include/exclude discovered nodes (details how to specify node filters are found in the `Elasticsearch documentation <https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster.html#cluster-nodes>`_) using the ``elasticsearch_discovery_filter`` setting, or tuning the frequency of the node discovery using the ``elasticsearch_discovery_frequency`` configuration option.
```

Comment:
This should be a link to the specific version of the Elasticsearch documentation:
https://www.elastic.co/guide/en/elasticsearch/reference/5.4/cluster.html#cluster-nodes